conf/shiro.ini.template

```diff
@@ -55,6 +55,33 @@ user3 = password4, role2
 #zeppelinHubRealm.zeppelinhubUrl = https://www.zeppelinhub.com
 #securityManager.realms = $zeppelinHubRealm
 
+### A sample for configuring OIDC Pac4J managed login(i.e. Keycloak)
```

Comment:
maybe we could separate config here in similar way as in documentation, by separating oidc, clients, callbackFilter configuration by new line